images/qbitmf/start.sh

```diff
@@ -16,8 +16,6 @@ chown abc:abc /downloads
 rm -rf /interface-state
 mkdir -p /interface-state
 
-python3 /make-wg-netns.py
```

Comment:
Any reason why this got moved down the script?

Reply:
To have the DNS configured in order to resolve wg endpoint hostname